action.yml

```
@@ -18,6 +18,12 @@ inputs:
   user-fraction:
     description: "Portion of users who should get the staged version of the app. Accepts values between 0.0 and 1.0 (exclusive-exclusive)"
     required: false
+  from-track:
+    description: "The source track to promote from. If not supplied, `beta` is used."
```

I'm not super familiar with Actions conventions, but would it make more sense to give this a default value and set required to true, rather than hide the default in the action code?

---

Good point - a default should be set: https://github.com/actions/toolkit/blob/main/packages/core/README.md#inputsoutputs, so I added it here instead.

I noticed the default is missing for `user-fraction` - should that be 1.0 or 0.0 in your mind? https://github.com/boswelja/AutoEvent/blob/main/.github/workflows/production-publish.yml#L15 doesn't have it... what does "undefined" do right now?